src/components/CreateInvitationForm.vue

```diff
@@ -225,13 +205,20 @@ export default defineComponent({
     onChangeApartment(apartmentId: number): void {
       this.apartmentId = apartmentId;
     },
+    correctType(): string {
+      const statusMap: any = {
+        ОСББ: 'cooperation',
+        Квартира: 'apartment',
```

Comment:
It's better to use only english keys in objects, but you can set values on multiple languages

Reply:
Updated))